src/jit-format/jit-format.cs

```diff
@@ -644,11 +693,11 @@ public static bool RunClangFormat(List<string> filenames, bool fix, bool verbose
 
                 if (verbose)
                 {
-                    Console.WriteLine("Running: clang-format {0} {1}", formatFix, filename);
+                    Console.WriteLine("Running: clang-format {0} -style=file {1}", formatFix, filename);
```

Comment:
Is this fix related?  If not, it should either be separated into its own change or accounted for in the description.